web/.storybook/preview.tsx

```diff
@@ -39,6 +40,9 @@ const preview: Preview = {
     layout: "fullscreen",
     controls: { expanded: true },
     actions: { argTypesRegex: "^on.*" },
+    docs: {
+      theme: themes.dark,
+    },
```

Comment:
In `theme.ts` we create our own theme. Can you use that with docs too?

Reply:
Updated.